circularprogressbar/src/main/java/com/jackandphantom/circularprogressbar/CircleProgressbar.java

```diff
@@ -94,6 +92,10 @@ public CircleProgressbar(Context context) {
      */
     public CircleProgressbar(Context context, AttrSet attrs) {
         this(context, attrs, 0);
+        initialize();
+    }
+
+    private void initialize() {
```

Comment:
initialize() can be called from the third constructor too. Please update.

Reply:
Done